src/Compilers/CSharp/Test/Emit/CodeGen/IndexAndRangeTests.cs

```diff
@@ -4236,5 +4214,170 @@ static void M(Span<int> s)
             var comp = CreateCompilation(source, targetFramework: TargetFramework.Net70);
             comp.VerifyDiagnostics();
         }
+
+        [Theory]
+        [InlineData(false)]
+        [InlineData(true)]
+        public void SingleOverloadReadOnlySpan(bool isMissing)
+        {
+
+            string source = """
+                using System;
+                
+                Console.Write(Util.SecondToLast("0123").ToString());
+
+                static class Util
+                {
+                    public static ReadOnlySpan<char> SecondToLast(ReadOnlySpan<char> s) => s[1..];
+                }
+                """;
+            var comp = CreateCompilationWithIndexAndRange(new[] { source, TestSources.GetSubArray, TestSources.Span, TestSources.MemoryExtensions, TestSources.ITuple },
+                                                   TestOptions.UnsafeReleaseExe);
+            if (isMissing)
+                comp.MakeMemberMissing(WellKnownMember.System_ReadOnlySpan_T__Slice_Int);
+            // If Verification.Skipped is not passed, the IL Verify will fail with:
+            //     System.Exception : IL Verify failed unexpectedly:
+            //     [SecondToLast]: Return type is ByRef, TypedReference, ArgHandle, or ArgIterator. { Offset = 0x8 }
```

Comment:
Instead of this, please use the `FailsILVerify.WithILVerifyMessage` helper. You can find examples in the ref struct interfaces tests.

Reply:
Done